### PR TITLE
ERMS-2660, ERMS-2661

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/CHANGELOG-TMP.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 **Ticket    Date    Branch      Author  Feature/Bug     Description/Keywords**
 
+2660    06.06.2020  rc1.4       Moe     Bug         API Endpunkt CostItemList muss alle CostItems zurückgeben
```
Comment (croqueGrec09) [inline app/CHANGELOG-TMP.md:5]: wrong date

File: app/src/groovy/de/laser/api/v0/entities/ApiCostItem.groovy [hunk 2]
```diff
@@ -136,6 +144,8 @@ class ApiCostItem {
         result.startDate           = ApiToolkit.formatInternalDate(costItem.startDate)
         result.taxRate             = costItem.taxKey?.taxRate ?: ((costItem.taxKey?.taxRate == 0) ? costItem.taxKey?.taxRate : costItem.taxRate)
 
+        result.isVisibleForSubscriber = costItem.isVisibleForSubscriber
```
Comment (klodav) [inline app/src/groovy/de/laser/api/v0/entities/ApiCostItem.groovy:147]: map boolean to 'Yes':'No'

File: app/web-app/swagger/v0/_schemas.gsp [hunk 3]
```diff
@@ -141,6 +141,8 @@
           format: <% print ApiToolkit.DATE_TIME_PATTERN %>
         finalCostRounding:
           type: string
+        isVisibleForSubscriber:
+          type: string
```
Comment (klodav) [inline app/web-app/swagger/v0/_schemas.gsp:145]:
```
isVisibleForSubscriber:
    type: string #mapped to boolean
    description: Mapping RefdataCategory "${RDConstants.Y_N}"
```

File: app/web-app/swagger/v0/changelog.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
 
 Source: ${grailsApplication.config.grails.serverURL} - Version: ${de.laser.api.v0.ApiManager.VERSION}
 
+#### 0.110
+
+- fiexed __/costItemList__
```
Comment (klodav) [inline app/web-app/swagger/v0/changelog.md:7]: - internal refactoring

File: app/web-app/swagger/v0/changelog.md.gsp [hunk 5]
```diff
@@ -2,6 +2,10 @@
 
 Source: ${grailsApplication.config.grails.serverURL} - Version: ${de.laser.api.v0.ApiManager.VERSION}
 
+#### 0.110
+
+- fiexed __/costItemList__
```
Comment (klodav) [inline app/web-app/swagger/v0/changelog.md.gsp:7]: - internal refactoring

**Pull request merged**